disperser/dataapi/operator_handler.go

```diff
@@ -229,18 +275,29 @@ func (s *OperatorHandler) ScanOperatorsHostInfo(ctx context.Context) (*SemverRep
 
 }
 
-func (oh *OperatorHandler) CreateOperatorQuorumIntervals(ctx context.Context, nonsigners []core.OperatorID, nonsignerAddressToId map[string]core.OperatorID, startBlock, endBlock uint32) (OperatorQuorumIntervals, []uint8, error) {
-	// Get operators' initial quorums (at startBlock).
+// CreateOperatorQuorumIntervals creates OperatorQuorumIntervals that are within the
+// the block interval [startBlock, endBlock] for operators specified in OperatorSet.
+//
+// Note: the returned result OperatorQuorumIntervals[op][q] means a sequence of increasing
+// and non-overlapping block intervals during which the operator "op" is registered in
+// quorum "q".
+func (oh *OperatorHandler) CreateOperatorQuorumIntervals(
+	ctx context.Context,
+	operatorSet *OperatorSet,
```

Comment:
Now this computes quorum intervals for operators of interest specified in this param, and it doesn't matter if they nonsigners or not